stdlib/2and3/syslog.pyi

```diff
@@ -35,4 +37,4 @@ def LOG_UPTO(a: int) -> int: ...
 def closelog() -> None: ...
 def openlog(ident: str = ..., logoption: int = ..., facility: int = ...) -> None: ...
 def setlogmask(x: int) -> int: ...
-def syslog(priority: int, message: str) -> None: ...
+def syslog(priority: Union[int, str], message: str = ...) -> None: ...
```

Comment:
Actually I just realized this is still not correct. It should be an overload for syslog(int, str) and syslog(str). (The current version also allows syslog(str, str), which is incorrect.)